docs/sources/setup-grafana/configure-security/configure-authentication/azuread/index.md

```diff
@@ -169,6 +169,18 @@ GF_AUTH_AZUREAD_CLIENT_SECRET
 
 **Note:** Verify that the Grafana [root_url]({{< relref "../../../configure-grafana/#root-url" >}}) is set in your Azure Application Redirect URLs.
 
+### Access token expiration check and configure refresh token
+
+> Available in Grafana v9.3 and later versions.
+
+> **Note:** This feature is behind the `accessTokenExpirationCheck` feature toggle.
+
+Grafana verifies that the User's access token is not expired in case the User is logged in using an OAuth provider and it tries to obtain a new access token by using the provided refresh token (if any) if the access token is expired.
+
+A refresh token is a special token that is used to get new access tokens without having to ask the user to log in again. Without a refresh token the User gets logged out from the system after the access token is expired.
```

Comment:
Unable to leave a suggestion for some reason. Maybe use this wording?

"Grafana uses a refresh token to obtain a new access token without requiring the user to log in again. If a refresh token doesn't exist, Grafana logs the user out of the system after the access token has expired."